NodeJS-server/handlers/errorHandler.js

```diff
@@ -0,0 +1,32 @@
+const logger = require('./logger');
+const HTTP_INTERNAL_SERVER_ERROR = 500;
+const MESSAGE = 'Internal Server Error';
+
+
+/**
+ * Express standard error middleware
+ *
+ * @param  {Error}      err     error object
+ * @param  {Object}     req     express request object
+ * @param  {Object}     res     express response object
+ * @param  {Function}   next    express next function
+ */
+let serverErrors = (err, req, res, next) => {
+  res.status(err.code || HTTP_INTERNAL_SERVER_ERROR).json(err.message || MESSAGE);
```

Comment:
Better to return object with error field. Try to think about it from service consumer perspective.